### PR TITLE
Cameracalibration 39 34

### PR DESCRIPTION
**Pull request by pejotejo**
## Why? What?

*Describe here why you created this PR and what it does*

Fixes #

## ToDo / Known Issues

*If this is a WIP describe which problems are to be fixed.*

## Ideas for Next Iterations (Not This PR)

*If there are some improvements that could be done in a next iteration, describe them here.*

## How to Test

*Describe how to test your changes. (For the reviewer)*

### CODE REVIEW

File: etc/parameters/rc24a/head.P0000074A09S28J00025.json
Comment (PatrickGoettsch) [inline etc/parameters/rc24a/head.P0000074A09S28J00025.json:1]: This file should be deleted

File: etc/parameters/rc24a/head.P0000074A09S31C00028.json
Comment (PatrickGoettsch) [inline etc/parameters/rc24a/head.P0000074A09S31C00028.json:1]: The file should be deleted

**Review by PatrickGoettsch — CHANGES_REQUESTED** [review]
Otherwise LGTM

File: etc/parameters/rc24a/head.P0000074A09S28J00025.json
Comment (pejotejo) [reply etc/parameters/rc24a/head.P0000074A09S28J00025.json:1]: done

**Review by PatrickGoettsch — APPROVED** [review]

**Pull request merged**